docs/release-notes/self-hosted/self-hosted-v3.2.0.md

```diff
@@ -0,0 +1,57 @@
+# Self-hosted v3.2.0
+
+These release notes are for [Codacy Self-hosted v3.2.0](https://github.com/codacy/chart/releases/tag/3.2.0){: target="_blank"}, released on December ##, 2020. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+## Product enhancements
+
+-   To overcome timeouts when loading the repositories list page on Codacy Self-hosted, it's now possible to configure how many repositories to load at a time. To do this, set a value lower than the default of 100 by configuring `codacy-spa.config.codacy.pagination.repositoriesLimit` on your `values-production.yml` configuration file. (CY-3352)
```

Comment:
@machadoit @jamescodacy, please check if the wording is appropriate.

I decided to include this in the "Product enhancements" because we ended up not actually fixing an issue, but instead adding more flexibility in the product to act as a workaround. But please let me know if you disagree.

Reply:
I like it! Maybe @heliocodacy can clarify on the `values-production.yml`, I think that the user usually has a values.yaml on their own right? What do we usually suggest to change on this cases?

I like the "To overcome timeouts" to avoid people messing with this value for free 😅

Reply:
Please check the most recent version instead, I think it describes the new capability a bit better. :+1:

Reply:
I think I prefer your original proposal, and maybe mentioning the “takes a very long time to load” may be a mistake on my part. Since the aim is the user with the timeout.

My concern now, is that some people consider “long time to load” too soon, and change this. And this value is tricky, because lower values also have other problems, like calling the provider more times

Reply:
Whoever is installing Codacy, downloads a copy of the values-production.yaml file and updates what makes sense there. If there are breaking changes on this file, user will download the new one and update it with the values that make sense like encryption keys and connection strings.

Reply:
Thanks for the extra context @heliocodacy! So this means that it makes sense to ask customers to change the new setting on their `values-production.yml` configuration file, right?